parking_permits/templates/emails/temporary_vehicle_deactivated.html

```diff
@@ -0,0 +1,19 @@
+{% extends "emails/base.html" %}
```

Comment:
I think we should use the same template as when normal vehicle details are updated here, so this one: https://app.abstract.com/share/20f2338b-e1bc-4dbc-bdad-109d98f0c559?collectionLayerId=a5375f85-2be8-46b4-b9d3-c841e0e930e9&mode=design&sha=fa21ac2338626295337ca35f601dcd4f566c1a28